lib/push.js

```diff
@@ -49,49 +53,75 @@ module.exports = function (log, db) {
      * @promise
      */
     notifyUpdate: function notifyUpdate(uid, reason) {
-      var events = reasonToEvents[reason] || reasonToEvents.accountVerify
+      reason = reason || 'accountVerify'
+      return this.pushToDevices(uid, reason)
+    },
+
+    /**
+     * Send a push notification with or without data to all the devices in the account (except the ones in the excludedDeviceIds)
+     *
+     * @param uid
+     * @param reason
+     * @param data
+     * @param excludedDeviceIds
+     * @promise
+     */
+    pushToDevices: function pushToDevices(uid, reason, data, excludedDeviceIds) {
+      var events = reasonToEvents[reason]
       return db.devices(uid).then(
         function (devices) {
           devices.forEach(function (device) {
             var deviceId = device.id.toString('hex')
 
+            if (excludedDeviceIds && excludedDeviceIds.indexOf(deviceId) !== -1) {
```

Comment:
It's be nice if we could use ES6 `Set` type here rather than doing a lookup in a list...but no big deal and it should always be a small list in practice